.github/vale/Vocab/Docker/accept.txt

```diff
@@ -24,3 +24,5 @@ Swarm Mode
 [Mm]oby
 dockerd
 dockerignore
+Docker Hub Vulnerability Scanning
```

Comment:
I'm not sure of the detailed impact in this vale config file ; but should we use "Docker Vulnerability Scanning", and not make this specific to Hub ?

Reply:
For now, it's probably easier I just add both, though the file does support regex, but as we will have minimal uses, I think just using the two entries is fine for now

Reply:
It's mostly here so that our style checkers ignore the "incorrect" capitalisation